src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,143 @@
+        defaultUser.setPassword(INITIAL_PASSWORD);
+        defaultUser.setAge(INITIAL_AGE);
+    }
+
+    @AfterEach
+    void after() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void register_nullArgument_notOk() {
+        assertThrows(InvalidRegistrationDataException.class, () -> {
+            registrationService.register(null);
+        }, "InvalidRegistrationDataException expected for null argument");
+    }
+
+    @Test
+    void register_loginIsNull_notOk() {
+        defaultUser.setLogin(null);
+        assertThrows(InvalidRegistrationDataException.class, () -> {
+            registrationService.register(defaultUser);
+        }, "InvalidRegistrationDataException expected for user.login = null");
+    }
+
+    @Test
+    void register_ageIsNull_notOk() {
+        defaultUser.setAge(null);
+        assertThrows(InvalidRegistrationDataException.class, () -> {
+            registrationService.register(defaultUser);
+        }, "InvalidRegistrationDataException expected for user.age = null");
+    }
+
+    @Test
+    void register_passwordIsNull_notOk() {
+        defaultUser.setPassword(null);
+        assertThrows(InvalidRegistrationDataException.class, () -> {
+            registrationService.register(defaultUser);
+        }, "InvalidRegistrationDataException expected for user.password = null");
+    }
+
+    @Test
+    void register_loginAlreadyInStorage_notOk() {
+        User storageUser = new User();
+        storageUser.setLogin(NON_UNIQUE_LOGIN);
+        Storage.people.add(storageUser);
+        defaultUser.setLogin(NON_UNIQUE_LOGIN);
+        assertThrows(InvalidRegistrationDataException.class, () -> {
+            registrationService.register(defaultUser);
+        }, InvalidRegistrationDataException.class.getName()
+                + ", method does not allow user login duplicates");
+    }
+
+    @Test
+    void register_loginIsEmpty_notOk() {
+        defaultUser.setLogin("");
+        assertThrows(InvalidRegistrationDataException.class, () -> {
+            registrationService.register(defaultUser);
+        }, "InvalidRegistrationDataException,"
+                + "when login field empty an exception should be thrown");
+    }
+
+    @Test
+    void register_default_ok() {
+        registrationService.register(defaultUser);
+        assertEquals(INITIAL_LOGIN, defaultUser.getLogin());
+        assertEquals(1, defaultUser.getId());
+        assertEquals(INITIAL_PASSWORD, defaultUser.getPassword());
+        assertEquals(INITIAL_AGE, defaultUser.getAge());
+    }
+
+    @Test
+    void register_passwordLengthLessThan6_notOk() {
+        defaultUser.setPassword("");
+        assertThrows(InvalidRegistrationDataException.class, () -> {
+            registrationService.register(defaultUser);
+        }, "InvalidRegistrationDataException,"
+                + "if password length less than 6 an exception should be thrown");
+    }
+
+    @Test
+    void register_AgeLessThan18_notOk() {
```

Comment:
```suggestion
    void register_ageLessThan18_notOk() {
```
let's follow the same naming pattern everywhere